### ISSUE TITLE
metrics type is changed from summary type to histogram type.

### ISSUE THREAD

**Issue by chenbiaoit** [open-telemetry/opentelemetry-java-instrumentation#9484]
### Describe the bug
percentile metrics was modified into a histogram

### Steps to reproduce

spring-boot yml config:
management:
  endpoints:
    web:
      exposure:
        include: "prometheus"
  metrics:
    distribution:
      percentiles:
        http:
          server:
            requests: 0.5,0.95

The above configuration will automatically generate the quantile index of http request calls.
After reporting the data to the opentelemetry collector through the agent, The metrics type is changed from summary type to histogram type., and the Metrics Name was also changed to http_server_requests_seconds_bucket. and lost the quantile tag

### Expected behavior

When I access /actuator/prometheus, I get the following metrics:

http_server_requests_seconds{application="pushgateway-test-boot-2.x",exception="None",instance="127.0.0.1:8080",method="GET",namespace="sit",outcome="SUCCESS",project="pushgateway-test-boot-2.x",status="200",uri="/updateSummary",quantile="0.5",} 0.003538944

### Actual behavior

the Metrics Type was changed to the histogram type, and the Metrics Name was also changed to http_server_requests_seconds_bucket.and lost the quantile tag！

### Javaagent or library instrumentation version

1.25.0

### Environment

JDK:8
Springboot:2.2.0.RELEASE

ageng config
-Dotel.metric.export.interval=15000
-Dotel.resource.attributes=service.name=puwhgateway-test-boot-2.x-test,service.namespace=XXX
-Dotel.exporter.otlp.endpoint=XXXX
-Dotel.exporter.otlp.traces.protocol=http/protobuf
-Dotel.exporter.otlp.metrics.protocol=http/protobuf
-Dotel.metrics.exporter=otlp
-Dotel.logs.exporter=none
-Dotel.instrumentation.micrometer.prometheus-mode.enabled=true

**Comment by chenbiaoit** [comment]
Just now I discovered that another metric was also generated, 
'http_server_requests_percentile_seconds' in which the percentile tag was modified to "phi"

**Comment by chenbiaoit** [comment]
This problem occurs due to the following code block, but why can't it be compatible with PrometheusMeterRegistry and directly perform HistogramGauges.registerWithCommonFormat? This will result in many additional indicators being generated and the percentile indicator being renamed

instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/OpenTelemetryMeterRegistry.java
`@Override
  protected DistributionSummary newDistributionSummary(
      Meter.Id id, DistributionStatisticConfig distributionStatisticConfig, double scale) {
    OpenTelemetryDistributionSummary distributionSummary =
        new OpenTelemetryDistributionSummary(
            id,
            config().namingConvention(),
            clock,
            distributionStatisticConfig,
            distributionStatisticConfigModifier,
            scale,
            otelMeter);
    if (distributionSummary.isUsingMicrometerHistograms()) {
      HistogramGauges.registerWithCommonFormat(distributionSummary, this);
    }
    return distributionSummary;
  }`

**Comment by mateuszrzeszutek** [comment]
Hey @chenbiaoit 

> ###
> the Metrics Type was changed to the histogram type, 

The `OpenTelemetryMeterRegistry` translates from micrometer concepts (`Timer`/`DistributionSummary`) to OpenTelemetry concepts (`DoubleHistogram`). This is by design; and there might be _some_ differences between Micrometer's Prometheus output and OpenTelemetry Prometheus output.

> and the Metrics Name was also changed to http_server_requests_seconds_bucket.and lost the quantile tag！

OTel does not natively support client-side percentiles; we do not export them by default. There is an option to add them back as series of gauges (kinda prometheus-style), you can enable it by setting `otel.instrumentation.micrometer.histogram-gauges.enabled` to `true`. 

> -Dotel.instrumentation.micrometer.prometheus-mode.enabled=true

This is a recommended setting only if you use the OTel Prometheus exporter; since you're using the OTLP metrics exporter I'd advice keeping it turned off. 

> This problem occurs due to the following code block, but why can't it be compatible with PrometheusMeterRegistry and directly perform HistogramGauges.registerWithCommonFormat? This will result in many additional metrics being generated and the percentile metrics being renamed

Because the `OpenTelemetryMeterRegistry` is a _general_ mapping from Micrometer to OpenTelemetry, and as I explained above OTel does not support some of the concepts used here.

**Comment by chenbiaoit** [comment]
Thank you very much for @mateuszrzeszutek's answer,
Because I used to use prometheus's SD function to collect metrics, and now I have encountered this kind of problem when migrating to the Otel-Agent solution. I read some literature and source code, but did not find the relevant answer. Based on this, if If possible, I would like to make an optimization suggestion here

How to smoothly and seamlessly migrate prometheus SD functions to OTel.
